projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1446,6 +1446,18 @@ function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
 					$info = (object) $metadata['videopress'];
 				}
 
+				if ( isset( $info->rating ) ) {
+					$response['rating'] = $info->rating;
+				}
+
+				if ( isset( $info->display_embed ) ) {
+					$response['display_embed'] = (string) (int) $info->display_embed;
+					// If not, default to metadata (for WPCOM).
+				} elseif ( isset( $metadata['videopress']['display_embed'] ) ) {
+					// We convert it to int then to string so that (bool) false to become "0".
+					$response['display_embed'] = (string) (int) $metadata['videopress']['display_embed'];
```

Comment:
Would be a ternary condition like below be less confusing than the double casting?

```suggestion
					$response['display_embed'] = $metadata['videopress']['display_embed'] ? '1' : '0';
```

Reply:
Personally I try to use explicit checks every time and try to avoid having conditions with checks like this because we are not always certain about the type of the value and PHP has some subtle cases that might turn to some unexpected behaviour, plus it's more forward compatible with possible changes in future PHP releases. 

Another reason for avoiding this (in general, in this case it might not be applicable) is that it basically it adds a duplicated validation logic in disguise just for formatting/outputting data that should be already in a valid state.